ansible/templates/etc/polkit-1/rules.d/00-deploy-static-sticky-service.rules.j2

```diff
@@ -0,0 +1,14 @@
+polkit.addRule(function(action, subject) {
+    polkit.log("action=" + subject);
+    polkit.log("subject=" + subject);
+    if (action.id == "org.freedesktop.systemd1.manage-units") {
+        if (action.lookup("unit") == "deploy-static-sticky.service") {
+        var verb = action.lookup("verb");
+        if (verb == "start" || verb == "stop" || verb == "restart") {
+            if (subject.user == "aas") {
+            return polkit.Result.YES;
```

Comment:
Nit: whitespace/indentation (same for the braces)

Reply:
are the changes I did satisfactory?